index.bs

```diff
@@ -10696,6 +10703,95 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
 </div>
 
+## The autofill Module ## {#module-autofill}
+
+The <dfn export for=modules>autofill module</dfn> contains functionality for triggering autofill.
+
+### Definition ### {#module-autofill-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+InputCommand = (
```

Comment:
InputCommand needs to be added to CommandData as well

Reply:
added AutofillCommand to CommandData, please let me know if this is how it was intended